src/TextArea/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import asInput, { inputProps } from '../asInput';
+
+function Text(props) {
```

Comment:
If you want to go down the syntactic sugar rabbit hole, you could write this as 

```javascript
const Text = props => (<textarea  ...props />)
```

and save a couple lines - obviously not a big deal either way